### PR TITLE
Security groups

### PR DESCRIPTION
**Pull request by atsikham**
Need to discuss:
* Should be default security group so open? 
* What should be the default value for security groups?
* Do we need examples how to pass security groups to a module?
* Is it necessary to pass a name in a variable for `aws_security_group` resource?
* Is suggested format ok now or it should be extended with additional available [parameters](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/security_group), such as `ipv6_cidr_blocks`?

### CODE REVIEW

**Review by erzetpe — DISMISSED** [review]

**Comment by mkyc** [comment]
Regarding PR main comment:

- no
- I believe that only SSH access
- probably
- it should be optionally possible
- I think it's good enough

**Comment by plirglo** [comment]
For now default configuration is too open for me.
From my perspective we should have restricted access to cluster from external (maybe except ssh port 22, which could be opened).
All communication between hosts inside the same cluster (same vpc) might be open for all ports (like we currently have in Epiphany).

We also add some notice / caution that changing this values too open might be risky and dangerous.

**Comment by atsikham** [comment]
@mkyc there is no straightforward way for optional map attributes, but since Terraform [0.14](https://github.com/hashicorp/terraform/releases/tag/v0.14.4) it's possible to do, this is an experimental [feature](https://www.terraform.io/docs/configuration/types.html#experimental-optional-object-type-attributes), we could use that together with [lookup](https://www.terraform.io/docs/configuration/functions/lookup.html) function.

Modified default value and readme with an example how to pass such complex parameters.

**Review by erzetpe — APPROVED** [review]

File: tests/integration_test.go [hunk 1]
```diff
@@ -64,7 +64,7 @@ func TestOnInitWithDefaultsShouldCreateProperFileAndFolder(t *testing.T) {
 	expectedFileContentRegexp := "kind: state\nawsbi:\n  status: initialized"
 
 	// when
-	_, stderr := runDocker(t, "init", "M_NAME="+moduleName)
+	_, stderr := runDocker(t, "init", "M_NAME="+moduleName, "M_REGION="+awsRegion)
```
Comment (mkyc) [inline tests/integration_test.go:67]: Shouldn't it be added to more places?

**Review by mkyc — APPROVED** [review]
Just one question regarding code

**Pull request merged**